### ISSUE TITLE
Feature Reqest: Item number column

### ISSUE THREAD

**Issue by TheFiZi** [JosefNemec/Playnite#2799]
### Feature description

Column that can be shown/hidden that displays the entry number in the list for each game like rows in Excel.

Sorry for the bad mock-up but I figured Excel was the easiest way to show that I'm meaning.

The number just needs to start at 1 and increment per game in the list.

My use case for this is I have a category with all my unplayed games. I can pick a random number between 1 and <TOTAL GAMES IN CATEGORY> and instead of having to count and press the down arrow on my keyboard to find the game I can just look at the number column and know which game is 38 (for example) in my current filtered view.

### Screenshots

![image](https://user-images.githubusercontent.com/1762426/152661064-6f5f1f74-a51a-4561-bef4-4fc888cd16bf.png)

**Comment by Jeshibu** [comment]
There is a random game picker that might make this request obsolete for you. It picks from your current list of games (so filters apply). The button is the one with a dice icon, or you can press F6.

**Comment by TheFiZi** [comment]
> There is a random game picker that might make this request obsolete for you. It picks from your current list of games (so filters apply). The button is the one with a dice icon, or you can press F6.

In my case I have friends pick a number between x and y and then end up having to count titles until I hit the number they gave me. Having the number displayed would save me the counting.

**Comment by JosefNemec** [comment]
I don't know about this. It's such a specific use case that I'm not sure it make sense adding it as built-in feature.

**Comment by TheFiZi** [comment]
> I don't know about this. It's such a specific use case that I'm not sure it make sense adding it as built-in feature.

Fair enough. Figured I'd ask. I assumed the number data was already there (iterating through a loop to display the data in the GUI) and it just needed to be made visible.
